apps/heft/src/templates/copy-static-assets.json

```diff
@@ -4,6 +4,10 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/heft/copy-static-assets.schema.json"
 
+  // Optionally specifies another JSON config file that this file extends from. This provides a way for standard
+  // settings to be shared across multiple projects.
+  // "extends": "base-project/config/copy-static-assets.json",
```

Comment:
@iclanton @D4N14L  Why is everything commented out?

Reply:
I had thought that because this is a template, it was just being used as an example, similar to how we comment out a lot in the default Rush templates